lib/common/global_header.dart

```diff
@@ -21,7 +23,7 @@ class GlobalHeader extends StatelessWidget implements PreferredSizeWidget {
     return AppBar(
       title: appBarTitle(), 
       actions: <Widget>[appBarAction(context, isVisible: showAction)],
-      backgroundColor: Colors.white,
+      backgroundColor: Colors.transparent,
```

Comment:
Changed to `transparent` because the white overwrites any lower-level background colours.